packages/core/echo/echo-pipeline/src/testing/test-data.ts

```diff
@@ -0,0 +1,127 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { DatabaseDirectory, ObjectStructure } from '@dxos/echo-protocol';
+import { getSchemaDXN } from '@dxos/echo-schema';
+import { DXN, ObjectId, PublicKey } from '@dxos/keys';
+
+import * as TestSchema from './test-schema';
+
+const spaceKey = PublicKey.random();
+
```

Comment:
better to have factories for these so that we can extend the tests and resuse the data. i don't like returning a named map.

```ts
const people = [{ name: 'Fred' }, { name: 'Alice' }].map(personFactory)

//
const alice = people.find(({ name }) => name === 'Alice');
```

next step.

Reply:
next pr